src/main/java/frc/robot/RobotContainer.java

```diff
@@ -174,47 +227,46 @@ private void configureBindings() {
         .whileTrue(
             Commands.either(intakeAmp(), intakeSpeaker(), robotStateManager.isAmpSupplier()));
 
-    OI.getTrigger(OI.Driver.outtake).whileTrue(intakeSubsystem.reverseIntakeCommand());
+    OI.getTrigger(OI.Driver.outtake).whileTrue(intakeCommandFactory.reverseIntakeCommand());
 
-    OI.getButton(OI.Driver.intakeSource).whileTrue(prepIntakeSource());
+    OI.getButton(OI.Driver.intakeSource).whileTrue(trapElvCommandFactory.wristintakeSource());
 
-    OI.getButton(OI.Driver.speakerSource).whileTrue(intakeSpeakerSource());
-  }
+    OI.getButton(OI.Driver.speakerSource).whileTrue(speakerSource());
+
+    OI.getButton(OI.Operator.prepClimb).onTrue(climberCommandFactory.raise());
 
-  private Command prepIntakeSource() {
-    return trapElvSubsystem
-        .intakeSource()
-        .until(trapElvSubsystem.getSourceBreak())
-        .andThen(trapElvSubsystem.intakeFromSourceForTime());
+    OI.getButton(OI.Operator.latchClimber).onTrue(climberCommandFactory.clip());
+
+    OI.getButton(OI.Operator.retractClimber).onTrue(climberCommandFactory.climb());
   }
 
-  private Command intakeSpeakerSource() {
-    return shooterSubsystem
-        .intakeSource()
-        .until(shooterSubsystem.getBeamBreak())
-        .andThen(shooterSubsystem.intakeSourceForTime());
+  private Command speakerSource() {
+    return Commands.deadline(
+        shooterCommandFactory.intakeSource(), triggerCommandFactory.getLoadCommand());
   }
 
   private Command intakeSpeaker() {
     return Commands.parallel(
-        intakeSubsystem.getSpeakerIntakeCommand(), triggerSubsystem.getLoadCommand());
+        intakeCommandFactory.getSpeakerIntakeCommand(), triggerCommandFactory.getLoadCommand());
   }
 
   private Command intakeAmp() {
-    return Commands.parallel(trapElvSubsystem.intakeGround(), intakeSubsystem.getAmpIntakeCommand())
-        .until(trapElvSubsystem.getSourceBreak())
-        .andThen(trapElvSubsystem.intakeFromGroundForTime());
+    return Commands.parallel(
+            trapElvCommandFactory.intakeGround(), intakeCommandFactory.getAmpIntakeCommand())
+        .until(trapElvCommandFactory.getSourceBreak())
+        .andThen(trapElvCommandFactory.intakeFromGroundForTime());
```

Comment:
making it so that the human doesn't need to handle timing stopping the intake. This intakes from the ground for then when detected intakes for a limited time longer.